viewing_party/party.py

```diff
@@ -1,23 +1,135 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None 
+    movie_dict = {}
+    if type(title) == str and type(genre) == str and type(rating) == int or type(rating) == float or int:

```

Comment:
we noticed after we submitted but the or int at the end of the line is not needed. 
```suggestion
    if type(title) == str and type(genre) == str and type(rating) == int or type(rating) == float or int:

```